pkg/core/serving/common/common.go

```diff
@@ -361,30 +381,181 @@ func parsePluginsCfg(s *openfunction.Serving, cm map[string]string, fc *function
 	return nil
 }
 
+func CreateDaprProxy(
+	ctx context.Context,
+	logger logr.Logger,
+	c client.Client,
+	scheme *runtime.Scheme,
+	s *openfunction.Serving,
+	cm map[string]string,
+	components map[string]*componentsv1alpha1.ComponentSpec,
+	componentName string) error {
+
+	labels := map[string]string{
+		OpenfunctionManaged: "true",
+		ProxyLabel:          s.Name,
+	}
+
+	selector := &metav1.LabelSelector{
+		MatchLabels: labels,
+	}
+
+	var port = constants.DefaultFuncPort
+	if s.Spec.Port != nil {
+		port = *s.Spec.Port
+	}
+
+	annotations := map[string]string{
+		DaprEnabled:         "true",
+		DaprAppID:           fmt.Sprintf("%s-%s", GetFunctionName(s), s.Namespace),
+		DaprLogAsJSON:       "true",
+		DaprAppProtocol:     "grpc",
+		DaprAppPort:         fmt.Sprintf("%d", port),
+		DaprListenAddresses: "[::],0.0.0.0",
+	}
+	annotations = util.AppendLabels(s.Spec.Annotations, annotations)
```

Comment:
I saw there is logic to append annotations, are we allowing users to append `labels` as well?